src/main/java/com/minecolonies/coremod/client/gui/WindowHireWorker.java

```diff
@@ -154,25 +148,25 @@ public void updateElement(final int index, @NotNull final Pane rowPane)
                     final AbstractBuildingWorker.Skill primary = ((AbstractBuildingWorker.View) building).getPrimarySkill();
                     final AbstractBuildingWorker.Skill secondary = ((AbstractBuildingWorker.View) building).getSecondarySkill();
 
-                    @NotNull final String strength = (primary.equals(AbstractBuildingWorker.Skill.STRENGTH) ? GREEN_STRING
-                                                        : (secondary.equals(AbstractBuildingWorker.Skill.STRENGTH) ? YELLOW_STRING : ""))
-                                                       + LanguageHandler.format("com.minecolonies.coremod.gui.citizen.skills.strength", citizen.getStrength()) + " §f ";
+                    @NotNull final String strength = (primary.equals(AbstractBuildingWorker.Skill.STRENGTH) ? ColorConstants.GREEN
+                                                        : (secondary.equals(AbstractBuildingWorker.Skill.STRENGTH) ? ColorConstants.YELLOW : ""))
+                                                       + LanguageHandler.format(COM_MINECOLONIES_COREMOD_GUI_CITIZEN_SKILLS_STRENGTH, citizen.getStrength()) + ColorConstants.WHITE;
 
-                    @NotNull final String charisma = (primary.equals(AbstractBuildingWorker.Skill.CHARISMA) ? GREEN_STRING
-                                                        : (secondary.equals(AbstractBuildingWorker.Skill.CHARISMA) ? YELLOW_STRING : ""))
-                                                       + LanguageHandler.format("com.minecolonies.coremod.gui.citizen.skills.charisma", citizen.getCharisma()) + " §f ";
+                    @NotNull final String charisma = (primary.equals(AbstractBuildingWorker.Skill.CHARISMA) ? ColorConstants.GREEN
+                                                        : (secondary.equals(AbstractBuildingWorker.Skill.CHARISMA) ? ColorConstants.YELLOW : ""))
+                                                       + LanguageHandler.format(COM_MINECOLONIES_COREMOD_GUI_CITIZEN_SKILLS_CHARISMA, citizen.getCharisma()) + ColorConstants.WHITE;
 
-                    @NotNull final String dexterity = (primary.equals(AbstractBuildingWorker.Skill.DEXTERITY) ? GREEN_STRING
-                                                         : (secondary.equals(AbstractBuildingWorker.Skill.DEXTERITY) ? YELLOW_STRING : ""))
-                                                        + LanguageHandler.format("com.minecolonies.coremod.gui.citizen.skills.dexterity", citizen.getDexterity()) + " §f ";
+                    @NotNull final String dexterity = (primary.equals(AbstractBuildingWorker.Skill.DEXTERITY) ? ColorConstants.GREEN
+                                                         : (secondary.equals(AbstractBuildingWorker.Skill.DEXTERITY) ? ColorConstants.YELLOW : ""))
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Extract this nested ternary operation into an independent statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS3358)